src/pages/trading-specification/components/_available-platform.tsx

```diff
@@ -74,55 +82,57 @@ const AvailablePlatform = () => {
                                             <Localize translate_text={deriv_x} />
                                         </StyledHeader>
                                     </Flex>
                                 </StyledLocalizedLink>
-                                <StyledLocalizedLink to="/derivez/">
-                                    <Flex direction="row">
-                                        <img src={DerivEZ} alt="Deriv EZ" width="24" height="24" />
-                                        <StyledHeader
-                                            type="paragraph-1"
-                                            weight="bold"
-                                            as="p"
-                                            width="auto"
-                                        >
-                                            <Localize translate_text={deriv_ez} />
-                                        </StyledHeader>
-                                    </Flex>
-                                </StyledLocalizedLink>
+                                {selected_market !== 'etfs' && (
+                                    <StyledLocalizedLink to="/derivez/">
+                                        <Flex direction="row">
+                                            <img
+                                                src={DerivEZ}
+                                                alt="Deriv EZ"
+                                                width="24"
+                                                height="24"
+                                            />
+                                            <StyledHeader type="paragraph-1" as="p" width="auto">
+                                                <Localize translate_text={deriv_ez} />
+                                            </StyledHeader>
+                                        </Flex>
+                                    </StyledLocalizedLink>
+                                )}
+                                {selected_market === 'etfs' && (
+                                    <StyledLocalizedLink to="/deriv-ctrader/">
+                                        <Flex direction="row">
+                                            <img
+                                                src={Ctrader}
+                                                alt="Deriv cTrader"
+                                                width="24"
+                                                height="24"
+                                            />
+                                            <StyledHeader type="paragraph-1" as="p" width="auto">
+                                                <Localize translate_text={deriv_ctrader} />
+                                            </StyledHeader>
+                                        </Flex>
+                                    </StyledLocalizedLink>
+                                )}
```

Comment:
```suggestion
            {
    isNotSelectedMarket ? (
        <>
            <StyledLocalizedLink to="/derivez/">
                <Flex direction="row">
                    <img src={DerivEZ} alt="Deriv EZ" width="24" height="24" />
                    <StyledHeader type="paragraph-1" as="p" width="auto">
                        <Localize translate_text={deriv_ez} />
                    </StyledHeader>
                </Flex>
            </StyledLocalizedLink>

            <StyledLocalizedLink to="/deriv-ctrader/">
                <Flex direction="row">
                    <img src={Ctrader} alt="Deriv cTrader" width="24" height="24" />
                    <StyledHeader type="paragraph-1" as="p" width="auto">
                        <Localize translate_text={deriv_ctrader} />
                    </StyledHeader>
                </Flex>
            </StyledLocalizedLink>
        </>
    ) : null
}
```

Reply:
and can define this condition to const  isNotSelectedMarket = selected_market !== 'etfs'